<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QMainWindow" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главная Страница - BlacklandRadio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>a dripping marker</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Blvcklvnd Rvdix</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="scroll">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>701</width>
      <height>391</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>697</width>
       <height>387</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="scrollLayout"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Новые Альбомы</string>
    </property>
   </widget>
   <widget class="QLabel" name="account_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Ваш аккаунт:</string>
    </property>
   </widget>
   <widget class="QLabel" name="account_value">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>accountname</string>
    </property>
   </widget>
   <widget class="QPushButton" name="my_profile_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Мой профиль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_album_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить альбом</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
